components/sections/supershardians/BeASuperShardians.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Button, Container, Flex, SimpleGrid, Text, VStack } from "@chakra-ui/react";
```

Comment:
Remove the components not being used.

![image](https://user-images.githubusercontent.com/7558499/168416390-9b2c5251-ff4c-425a-8ba0-f22d03d17da4.png)